test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationsHelper.java

```diff
@@ -22,4 +28,57 @@ public ConversationHelper goConversation() {
 
         return new ConversationHelper(new HelperSecret());
     }
+
+    public ConversationHelper goGroup() {
+        try {
+            this.assertText(groupName);
+            onView(withText(groupName)).perform(click());
+        } catch (NoMatchingViewException e) {
+            newGroup();
+        }
+
+        finally {
+            return new ConversationHelper(new HelperSecret());
+        }
+    }
+
+    public ConversationHelper goGroup(String groupname) {
+        try {
+            this.assertText(groupname);
+            onView(withText(groupname)).perform(click());
+        } catch (NoMatchingViewException e) {
+            newGroup();
+        }
+
+        finally {
+            return new ConversationHelper(new HelperSecret());
+        }
+    }
+
+   private void newGroup() {
+        openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getTargetContext());
+
+        onView(withText(R.string.text_secure_normal__menu_new_group))
```

Comment:
could you either put a space in between each one of these, or not put a space between any of them? it looks nicer if it's consistent.